tracer/test/snapshots/Security.AspNetCore5AsmRemoteRules._.verified.txt

```diff
@@ -75,7 +75,6 @@
       network.client.ip: 127.0.0.1,
       runtime-id: Guid_1,
       span.kind: server,
-      _dd.appsec.event_rules.version: 2.22.222,
```

Comment:
Was this change expected?

Reply:
Was this the result of the Regex updates in `AspNetBase.cs`?

Reply:
Yes, due to the regex changes. It's maybe original test author indented but I currently of the opinion that there are too man place where test that the event rule version is added correctly.